application_sandbox/overlapping_frames/main.cpp

```diff
@@ -480,6 +493,10 @@ int main_entry(const entry::EntryData* data) {
     uint32_t current_frame = 0;
     uint32_t next_frame = 1;
     uint32_t image_index;
+    std::chrono::steady_clock::time_point start_time_point =
+        std::chrono::steady_clock::now();
+    float triangle_speed = 0.008333333f;
```

Comment:
Do you precisely need `0.0083333` ? This looks like a "magic value", would `0.008` do a good-enough job?

Using `0.008`, or even `0.01`, would look more like an arbitrary choice, reflecting the fact that this variable can be set to various values around that range, and doesn't have to be a specific precise value like `0.00833333`.

In other words, if in some code you see `foo = 123.456789`, you might think "Oh, it looks like they had to fine-tune this value to some precise number", whereas if you see `foo = 100`, it feels more like "OK, foo should be roughly around value 100, but we might try some other values there".

Reply:
Also, make it a `const` if you can.

Reply:
Yeah, this value is not fine tuned. I just chose a desired speed on the basis on personal preference. I'll document this fact in the code.

Reply:
Cool. Any code update to this PR ?

Reply:
Sorry for the delay. It had slipped my mind. Are these new values alright?

Reply:
All good, thanks.